pages.pt_BR/linux/chage.md

```diff
@@ -5,20 +5,20 @@
 
 - Exibir as informações referentes a senha do usuário:
 
-`chage -l {{nome_do_usuario}}`
+`chage --list {{nome_do_usuario}}`
 
 - Habilitar a expiração da senha do usuário em 10 dias:
 
-`sudo chage -M {{10}} {{nome_do_usuario}}`
+`sudo chage --maxdays {{10}} {{nome_do_usuario}}`
 
 - Desabilitar a expiração da senha do usuário:
 
-`sudo chage -M -1 {{nome_do_usuario}}`
+`sudo chage --maxdays {{-1}} {{nome_do_usuario}}`
```

Comment:
The `-1` is not something that the user can change without changing the meaning of the example, so it shouldn't be in `{{token_syntax}}`.

Reply:
This has come up a couple of the times in the repo historically.
Whenever I've seen this commented/discussed, the conclusion has always been arguments should be wrapped in `{{}}`.

> > Sorry about that, I didn't realize we still had to use `{{...}}` if the command literally specifies an argument value.
> 
> No problem. Basically, everything which isn't a flag, argument or subcommand should be in `{{ }}`. Also it makes the page more readable. For comparison:
> ![image](https://user-images.githubusercontent.com/50295997/117638178-dc034a00-b182-11eb-94b3-863d055faa19.png)
> ![image](https://user-images.githubusercontent.com/50295997/117638133-d279e200-b182-11eb-866c-1a322c48e19b.png)
> 
> — https://github.com/tldr-pages/tldr/pull/5921#issuecomment-836462426

Reply:
I think we should use token syntax. Token syntax doesn't automatically mean that we value can't be changed. It should just make it easier to differentiate between flags and arguments (In my opinion). In the example above with `anbox` it's the same.

Reply:
Hmmm, interesting. I've always viewed it as something the user can change.

Reply:
For me it's always to highlight the argument options like enums, or this int values.

Reply:
In a case like this I often like to see which one looks more intuitive, but I'm actually not really sure. Maybe this would be worth opening a new issue to discuss what the purpose of the token syntax should be.
![image](https://user-images.githubusercontent.com/50295997/133886622-c0b58de2-df39-4a97-831e-ee8fcce20d48.png)
![image](https://user-images.githubusercontent.com/50295997/133886653-e1319a26-e6dd-4471-88e4-8ddc7ee5d28c.png)

Reply:
To be honest, I think in this case it's _much_ better to use the token syntax for `-1`.

If we didn't, it'd look to me that:

| Argument | Value
|---|---
|`--maxdays` | N/A
| `-1` | `username`

Without highlighting it, it looks like an argument, not the value to `--maxdays`.

Reply:
Hmm, I do see that there's a  beenfit to the token syntax here.